components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__genes.xml

```diff
@@ -137,10 +137,12 @@
         var infoTool = new Element('span', {'class' : 'fa fa-info-circle xHelpButton gene-info', 'title' : geneNames[i]});
         var icon_helpController = new PhenoTips.widgets.HelpButton(infoTool);
         var triangle = new Element('span', {'class': "fa fa-exclamation-triangle", title: '$services.localization.render('phenotips.patientSheetCode.termSuggest.nonStandardGenesymbol')'}).update(' ');
-        if (isSuggestSelected)
+        if (isSuggestSelected) {
```

Comment:
The name pretty much reflects the context. If suggested term is selected then the info icon is inserted. If not - the triangle icon. It has nothing to do with view mode. I put brackets there just for code improvement, this is not the actual bugfix. The actual bugfix happens in lines 171-173 where I restrict the whole function that generates those icons to be implemented only in view mode.